component/common.libsonnet

```
@@ -0,0 +1,45 @@
+local kap = import 'lib/kapitan.libjsonnet';
+local kube = import 'lib/kube.libjsonnet';
+local inv = kap.inventory();
+// The hiera parameters for the component
+local params = inv.parameters.appuio_cloud_reporting;
+
+local labels = {
+  'app.kubernetes.io/name': 'appuio-cloud-reporting',
+  'app.kubernetes.io/managed-by': 'commodore',
+  'app.kubernetes.io/part-of': 'syn',
+};
+
+local cronJob = function(name, schedule, jobSpec) kube._Object('batch/v1', 'CronJob', name) {
```

Why not use `kube.CronJob()`? Does OpenShift complain about `batch/v1beta1`?

```suggestion
local cronJob = function(name, schedule, jobSpec) kube.CronJob(name) {
```